docs/src/manual/nonlinear.md

```diff
@@ -556,6 +556,22 @@ f(x::Vector) = sum(x[i]^i for i in 1:length(x))
 @objective(model, Min, op_f(x...))
 ```
 
+If the operator takes several vector inputs, write a function which takes the
```

Comment:
```suggestion
If the operator takes several vector inputs, write a function that takes the
```

Reply:
My understanding is that we use "that" when it's a statement that additionally constrains the work and to use "which" when it is another information about it but it doesn't change the set of possibilities.